src/main/java/edu/umich/tl/analytics/CanvasCourseReportServlet.java

```diff
@@ -173,6 +192,29 @@ private void getThePublishedCourseList(String url, CoursesForSubaccounts cfs) {
 		}
 	}
 
+	//{"errors":[{"message":"The specified resource does not exist."}],"error_report_id":479313}
+
+	private String errorHandler(HttpResponse httpResponse, String url) {
```

Comment:
A more specific name would be good since there are errors that this doesn't handle.  Maybe apiCallErrorHandler ?